src/components/ContentHeader/ContentHeader.test.js

```diff
@@ -0,0 +1,19 @@
+import ContentHeader from "./ContentHeader";
+import { render, screen } from "@testing-library/react";
+
+describe("initial tests for content header", () => {
+
+    it("should render a heading and a link on the page", () => {
```

Comment:
Once you have added the link component you will need to update your tests. They will need to use the `customRender` function. Look at the navigation tests.

Reply:
A passing test has now been added, currently, only a single card component is loaded but it does confirm that passing the route through the prop works. The test can be updated to use the component to show all pathways once it is made.